x-pack/plugin/core/src/javaRestTest/java/org/elasticsearch/xpack/core/DataStreamRestIT.java

```diff
@@ -65,21 +71,29 @@ public void testDSXpackUsage() throws Exception {
         Request rollover = new Request("POST", "/logs-mysql-default/_rollover");
         client().performRequest(rollover);
 
+        // Create failure store data stream
+        indexRequest = new Request("POST", "/fs/_doc");
+        indexRequest.setJsonEntity("{\"@timestamp\": \"2020-01-01\"}");
+        client().performRequest(indexRequest);
+
         dataStreams = (Map<?, ?>) getLocation("/_xpack/usage").get("data_streams");
         assertNotNull(dataStreams);
         assertTrue((boolean) dataStreams.get("available"));
         assertTrue((boolean) dataStreams.get("enabled"));
-        assertThat("got: " + dataStreams, dataStreams.get("data_streams"), equalTo(1));
-        assertThat("got: " + dataStreams, dataStreams.get("indices_count"), equalTo(2));
+        assertThat("got: " + dataStreams, dataStreams.get("data_streams"), equalTo(2));
+        assertThat("got: " + dataStreams, dataStreams.get("indices_count"), equalTo(3));
+        failureStoreStats = (Map<String, Integer>) dataStreams.get("failure_store");
+        assertThat(failureStoreStats.get("enabled_count"), anyOf(equalTo(null), equalTo(1)));
```

Comment:
This should be:
```suggestion
        assertThat(failureStoreStats.get("enabled_count"), equalTo(1));
```
right?